core/server/master/src/main/java/alluxio/master/file/meta/InodeTree.java

```diff
@@ -1117,6 +1129,7 @@ public void syncPersistNewDirectory(MutableInodeDirectory dir)
       Long lastModificationTime = status.getLastModifiedTime();
       if (lastModificationTime != null) {
         dir.setLastModificationTimeMs(lastModificationTime, true);
+        dir.setLastAccessTimeMs(lastModificationTime, true);
```

Comment:
why does this depend on `lastModificationTime != null`?

Reply:
Currently UFS last access time is not exposed or sync'ed.  This is just to give last access time a default value.

Reply:
Added.